api/scripts/wazuh-apid.py

```diff
@@ -15,6 +15,8 @@
 from wazuh.core import pyDaemonModule
 
 SSL_DEPRECATED_MESSAGE = 'The `{ssl_protocol}` SSL protocol is deprecated.'
+CACHE_DEPRECATED_MESSAGE = 'The cache API configuration option was deprecated in {release} and will be removed in the ' \
+                           'next major.'
```

Comment:
```suggestion
CACHE_DEPRECATED_MESSAGE = 'The `cache` API configuration option was deprecated in {release} and will be removed in the ' \
                           'next major release.'
```

Reply:
Fixed in commit b3668e8263fe0eccdd4d6da019d3d55d4bcd170c.